x-pack/legacy/plugins/fleet/public/pages/agent_list/components/enrollment_instructions/index.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiSteps, EuiText, EuiCodeBlock } from '@elastic/eui';
+import { Writer } from 'mustache';
+
+export { ShellEnrollmentInstructions } from './shell';
+export { ContainerEnrollmentInstructions } from './container';
+export { ToolsEnrollmentInstructions } from './tools';
+
+export type ManualEnrollmentInstructions = Array<{
+  title: string;
+  textPre?: string;
+  commands?: string;
+  commandsLang?: 'bash' | 'yaml';
+}>;
+
+export const ManualEnrollmentSteps: React.SFC<{ instructions: ManualEnrollmentInstructions }> = ({
+  instructions,
+}) => (
+  <EuiSteps
+    steps={instructions.map(({ title, textPre, commands, commandsLang }) => ({
+      title,
+      children: (
+        <EuiText size="s">
+          {textPre ? <p>{textPre}</p> : null}
+          {commands ? (
+            // TODO: Increase overflowHeight when https://github.com/elastic/eui/issues/2435 is fixed
+            // or be smarter with setting this number before release
+            <EuiCodeBlock overflowHeight={150} language={commandsLang || 'bash'} isCopyable>
+              {replaceTemplateStrings(commands.trim())}
+            </EuiCodeBlock>
+          ) : null}
+        </EuiText>
+      ),
+    }))}
+  />
+);
+
+// Setup for replacing template variables in install instructions
+const TEMPLATE_TAGS = ['{', '}'];
```

Comment:
any reason to not use the default from mustache `{{ variable }}`?

Reply:
no particular reason, this syntax was copied from Kibana's Add Data setup which just uses single braces. which do y'all prefer?

Reply:
Personal preference of using the standard, less code is better code :)